src/cosmos/CosmosTools.ts

```diff
@@ -53,11 +56,22 @@ export class CosmosTools implements EdgeCurrencyTools {
     )
     this.methods = methods
     this.registry = registry
+    const { data, name, url } = this.networkInfo.chainData
+    this.chainData = data
```

Comment:
This is inconsistent naming. I'd recommend renaming `this.networkInfo.chainData` to `this.networkInfo.chainInfo`